package.json

```diff
@@ -63,6 +63,6 @@
     "typedoc-plugin-external-module-name": "^4.0.6",
     "typescript": "^4.2.2"
   },
-  "version": "0.24.0-1",
+  "version": "0.24.1-rc.1",
```

Comment:
Q: was this already published as this version?

Reply:
https://github.com/KILTprotocol/sdk-js/commit/497f1da52fd37b7a5d6d17a561d8601ba07173b7